src/openzaak/components/autorisaties/forms.py

```diff
@@ -445,11 +446,15 @@ def _validate_overlapping_types(self):
         scope_types = {}
         for form in self.forms:
             if form.cleaned_data:
-                types = set(form.get_types() or [])
+                component = form.cleaned_data["component"]
+                types = set(form.get_types(component) or [])
                 scopes = form.cleaned_data["scopes"]
-                types_field = COMPONENT_TO_FIELDS_MAP[form.cleaned_data["component"]][
-                    "types_field"
-                ]
+
+                _field_info = COMPONENT_TO_FIELDS_MAP.get(component)
+                if _field_info is None:
+                    continue
```

Comment:
I think if there are no types we still should validate scopes for overlapping or in this case duplicating each other.

Reply:
This function check for overlapping types. How can you check for types if there's no types field?

The rest of the for-loop continues right?